colorcoder/enums/MinorColor.java

```diff
@@ -0,0 +1,30 @@
+package colorcoder.enums;
+
+public enum MinorColor implements Color {
+    BLUE(0),
+    ORANGE(1),
+    GREEN(2),
+    BROWN(3),
+    SLATE(4);
+
+    private int index;
+
+    private MinorColor(int index) {
+        this.index = index;
+    }
+
+    @Override
+    public int getIndex() {
+        return index;
+
+    }
+
+    public static MinorColor fromIndex(int index) {
```

Comment:
This is very similar to the MajorColor fromIndex. Think of isolating common functionality and re-using it.